python/lsst/afw/fits/SConscript

```diff
@@ -1,3 +1,3 @@
 ## -*- python -*-
 #from lsst.sconsUtils import scripts
-#scripts.BasicSConscript.python(['fitsLib'])
+#scripts.BasicSConscript.pybind11(['fits'])
```

Comment:
I think the aim is to remove the commented out code when the pybind11 port is about to be merged to master itself. This commented out code will not be retained.